utils.js

```diff
@@ -20,6 +20,9 @@ const makePromise = () => {
 };
 
 function parseQuery(queryString) {
+  if(!queryString){
```

Comment:
Why do we need early return?

Reply:
If it's not a string coming in, it's a type error and we should fix the bug. If this is a string then the below code should be fine.

Reply:
Resolved it by adding the check in the route.